pkg/certificate/manager.go

```diff
@@ -129,15 +129,23 @@ func (m *Manager) handleMRCEvent(mrcClient MRCClient, event MRCEvent) error {
 		c := &issuer{Issuer: client, ID: clientID, CertificateAuthority: ca}
 		switch {
 		case mrc.Status.State == constants.MRCStateActive:
+			m.mu.Lock()
 			m.signingIssuer = c
 			m.validatingIssuer = c
+			m.mu.Unlock()
 		case mrc.Status.State == constants.MRCStateIssuingRollback || mrc.Status.State == constants.MRCStateIssuingRollout:
+			m.mu.Lock()
 			m.signingIssuer = c
+			m.mu.Unlock()
 		case mrc.Status.State == constants.MRCStateValidatingRollback || mrc.Status.State == constants.MRCStateValidatingRollout:
```

Comment:
I believe if a MRC is in a state of IssuingRollback it should only be used for validating.  I define the IssuingRollback state as the rollback of using this issuer to sign certificates.
```suggestion
		case mrc.Status.State == constants.MRCStateIssuingRollback || mrc.Status.State == constants.MRCStateValidatingRollout:
```

Reply:
The IssuingRollback state is set when an MRC enters the issuingRollout state. I don't think there should ever be a case where MRCs have the IssuingRollback and ValidatingRollout states.

Reply:
This comment is not blocking and can be addressed in a follow up PR.

Reply:
Hmmm so what does it mean of one MRC is active, and the other is ValidatingRollout? Keep in mind that the control plane could restart between status updates of two different MRCs